Server/index.js

```diff
@@ -36,7 +36,7 @@ import mjml2html from "mjml";
 
 // Settings Service and dependencies
 import SettingsService from "./service/settingsService.js";
-import AppSettings from "../db/models/AppSettings.js";
+import AppSettings from "./db/models/AppSettings.js";
```

Comment:
@ajhollid for some reason the two dots were still here, even though it has already been merged

Reply:
Wierd, it's in the codebase on `develop` branch now, perhaps this branch was created from develop before the commit was merged?  Anyway no big deal!